tests/api/test_mod_ref_type.py

```diff
@@ -38,40 +38,49 @@ def test_get_bad_mrt(self):
         assert test_mod_ref_type.response.status_code == status.HTTP_201_CREATED
         # check db for mrt
-        mrt = db.query(ModReferenceTypeModel).filter(
-            ModReferenceTypeModel.mod_reference_type_id == test_mod_ref_type.new_mod_ref_type_id).one()
-        assert mrt.reference_type == "string1"
-        assert mrt.reference.curie == test_mod_ref_type.related_ref_curie
-        assert mrt.source == "string2"
-
-    def test_patch_mrt(self, test_mod_ref_type, test_reference2, auth_headers): # noqa
+        rmrt = db.query(ReferenceModReferenceTypeAssociationModel).filter(
+            ReferenceModReferenceTypeAssociationModel.reference_mod_referencetype_id == test_mod_ref_type
+            .new_mod_ref_type_id).one()
+        assert rmrt.mod_referencetype.referencetype.label == "Journal"
+        ref_curie = db.query(ReferenceModel.curie).filter(
+            ReferenceModel.reference_id == rmrt.reference_id).one_or_none()[0]
+        assert ref_curie == test_mod_ref_type.related_ref_curie
+        assert rmrt.mod_referencetype.mod.abbreviation == "ZFIN"
+
+    def test_patch_mrt(self, db, test_mod_ref_type, test_reference2, auth_headers): # noqa
         with TestClient(app) as client:
             patch_data = {
                 "reference_curie": test_reference2.new_ref_curie,
-                "reference_type": "string3",
-                "source": "string4"
+                "reference_type": "Review",
+                "source": "ZFIN"
             }
             response = client.patch(url=f"/reference/mod_reference_type/{test_mod_ref_type.new_mod_ref_type_id}",
                                     json=patch_data, headers=auth_headers)
             # check db for mrt
             assert response.status_code == status.HTTP_202_ACCEPTED
             response = client.get(url=f"/reference/mod_reference_type/{test_mod_ref_type.new_mod_ref_type_id}")
             mrt = response.json()
-            assert mrt["reference_type"] == "string3"
+            assert mrt["reference_type"] == "Review"
             assert mrt["reference_curie"] == test_reference2.new_ref_curie
-            assert mrt["source"] == "string4"
+            assert mrt["source"] == "ZFIN"
 
             from_id = client.get(url=f"/reference/{test_mod_ref_type.related_ref_curie}").json()["reference_id"]
             to_id = client.get(url=f"/reference/{test_reference2.new_ref_curie}").json()["reference_id"]
 
             response = client.get(url=f"/reference/mod_reference_type/{test_mod_ref_type.new_mod_ref_type_id}/versions")
             transactions = response.json()
             assert transactions[0]['changeset']['reference_id'][1] == from_id
-            assert transactions[0]['changeset']['reference_type'][1] == "string1"
-            assert transactions[0]['changeset']['source'][1] == "string2"
+            mod_referencetype_id_orig = db.execute("SELECT mod_referencetype_id from mod_referencetype where mod_id = "
```

Comment:
making the SELECT lowercase here and in 79 might be more consistent, but also not important.